src/ngraph/runtime/cpu/CMakeLists.txt

```diff
@@ -157,5 +149,12 @@ if (NGRAPH_CPU_ENABLE)
     target_include_directories(cpu_backend SYSTEM PUBLIC libmkldnn)
     set_target_properties(cpu_backend PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${NGRAPH_BUILD_DIR})
 
+    if(NGRAPH_DISTRIBUTED_ENABLE)
+        add_definitions(-DNGRAPH_DISTRIBUTED)
```

Comment:
Use `target_compile_definitions` instead. It is cleaner
https://cmake.org/cmake/help/v3.0/command/target_compile_definitions.html
This is in other parts of the cmake too. I've been meaning to address it...

Reply:
Done